lib/spack/docs/getting_started.rst

```diff
@@ -1560,6 +1561,15 @@ As Spack is a Python-based package, an installation of Python will be needed to
 Python 3 can be downloaded and installed from the Windows Store, and will be automatically added
 to your ``PATH`` in this case.
 
+Additionally, the python module ``pywin32`` and ``clingo`` are required but not always installed from the Windows Store.
```

Comment:
We should remove any/all references to the Windows store python. It is fraught with issues and generally, I'd prefer we not promote one specific method of Python installation, particularly not this one.
 
These modules will not come bundled with Python, but are not requirements to run Spack, they'll both be installed on startup. 

We actually might not install `pywin32` on startup for the powershell scripts, but that should be fix by enabling that script to install pywin32 or making it part of the bootstrap process.

Reply:
I only mention it because it is the way windows advertises getting python. However, if we are bootstrapping this correctly now then probably not an issue. I would rather fix where it is missing in bootstrapping than document it like this.

Reply:
>  only mention it because it is the way windows advertises getting python

If anything this makes me want to add documentation actively discouraging that approach. 

> I would rather fix where it is missing in bootstrapping than document it like this.

Agreed. For now I'll add it to the powershell scripts with the eventual goal of bootstrapping via spack's actual bootstrapping mechanism.